### ISSUE TITLE
[Feature Request]: Enable mbregex option for mbstring extension

### ISSUE THREAD

**Issue by aaronbushnell** [beyondcode/herd-community#5]
### Feature Description

It looks like the `mbstring` extension is installed, but `mbregex` is not enabled which is required for Yii/Craft CMS projects. Is it possible to enable this option in the binaries supplied by Herd?

### Is this feature valuable for other users as well and why?

I think this feature would be valuable for other users who might be running Yii/Craft CMS projects in Herd as an option for those who might be using Valet, but would like something simpler.

**Comment by jacobgraf** [comment]
I just ran across this as well. Love Herd, but have a lot of Craft sites and would like to be able to use them with Herd as well. Here is the error I am getting.

```
Fatal error: Uncaught Error: Call to undefined function mb_regex_encoding() in /Users/jacobgraf/Sites/sommersconst/vendor/voku/portable-utf8/src/voku/helper/UTF8.php:536 Stack trace: #0 /Users/jacobgraf/Sites/sommersconst/vendor/voku/portable-utf8/bootstrap.php(7): voku\helper\UTF8::checkForSupport() #1 /Users/jacobgraf/Sites/sommersconst/vendor/composer/autoload_real.php(78): require('/Users/jacobgra...') #2 /Users/jacobgraf/Sites/sommersconst/vendor/composer/autoload_real.php(61): composerRequire70dc20ad819adea2a4bacfcda7153cde('b46ad4fe52f4d18...', '/Users/jacobgra...') #3 /Users/jacobgraf/Sites/sommersconst/vendor/autoload.php(7): ComposerAutoloaderInit70dc20ad819adea2a4bacfcda7153cde::getLoader() #4 /Users/jacobgraf/Sites/sommersconst/web/index.php(11): require_once('/Users/jacobgra...') #5 /Applications/Herd.app/Contents/Resources/valet/server.php(110): require('/Users/jacobgra...') #6 {main} thrown in /Users/jacobgraf/Sites/sommersconst/vendor/voku/portable-utf8/src/voku/helper/UTF8.php on line 536
```

**Comment by mpociot** [comment]
Yes, I'll publish a PHP update that includes it next week! 👍

**Comment by aaronbushnell** [comment]
Thank you, @mpociot! _Amazing_ work on this! 🚀

**Comment by mpociot** [comment]
PHP 8.2 with mbregex is now available. The other PHP versions are still compiling.
You can update via the Settings screen!

**Comment by jacobgraf** [comment]
Tested 8.2 with Craft and works like a dream! I'll keep an eye out for the other version! Thanks again man!

**Comment by jacobgraf** [comment]
The only thing failing now in the Craft System Report is `ini_set calls` which I'm not sure is a big deal or not. I've increased both `memory_limit` and `max_execution_time` but the `ini_set_calls` requirement still shows an error.

![image](https://github.com/beyondcode/herd-community/assets/146333/cbf89fe4-f4fa-4f51-ba66-a554c95f4b9d)

**Comment by jacobgraf** [comment]
@mpociot Did 8.0 get skipped for some reason? Of course, this is the version I need to match production for most of my current Craft sites 🤣

**Comment by jacobgraf** [comment]
@mpociot You the man! Saw the update for 8.0 and works like a charm! 👏🏻

**Comment by mpociot** [comment]
Sorry, took some time to compile 'em all :)

**Comment by knufri** [comment]
Hello together,
when I use php 8.1 with my Craft installations I get still the `Call to undefined function mb_regex_encoding()` error. I have updated all the php versions 🤔 Did I miss something, @jacobgraf can you use 8.1 with Craft?
Kind regards!

**Comment by mpociot** [comment]
Could you tell me the output of "php81 -v"?

**Comment by knufri** [comment]
Hello @mpociot, here the is the requested output:
```
❯ php81 -v
PHP 8.1.21 (cli) (built: Jul  7 2023 00:24:44) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.21, Copyright (c) Zend Technologies with Zend OPcache v8.1.21, Copyright (c), by Zend Technologies
```

**Comment by mpociot** [comment]
@knufri I helped someone debug a similar issue earlier today where they re-installed Herd and because of that the updater was thinking they were already on the latest version.

Please run the following via Terminal and then restart Herd. This should give you the 8.1 update again:

```
defaults write de.beyondco.herd installed_internal_8.1 1
```

**Comment by knufri** [comment]
Hello @mpociot thanks for that fast reply. Now it works also with php 8.1 - but with 8.3 the `mb_regex_encoding()` error persists. Is there any command like above I can try like `defaults write de.beyondco.herd installed_internal_8.3 1` ?

**Comment by aaronbushnell** [comment]
FWIW I never got the PHP 7.4 update until I ran:
```
defaults write de.beyondco.herd installed_internal_7.4 1
```
And restarted Herd. But that's fixed now! I presume 8.3 isn't updated due to it being an alpha, but I could be wrong.
